### PR TITLE
Deprecate --host and --port in favour of --bind-addr

### PR DESCRIPTION
**Pull request by nhooyr**

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -3,6 +3,6 @@
 build
 dist*
 out*
-release*
+release/
```
[the remark this answers is not in the document]
Comment (nhooyr) [reply .gitignore:6]: So my issue was `ci/release-image` was getting .gitignored. Didn't release we had `release-upload`. I'll git ignore both for now.

**Comment by antofthy** [comment]
So exactly what changed?

**Comment by nhooyr** [comment]
@antofthy If you run a prerelease, you can run `--help` to see the `--bind-addr` option and its description. It just combines --host and --port as described in the original issue. When we create a new release, we'll be sure to mention this option in the notes.

**Comment by antofthy** [comment]
Thank you.

**Pull request merged**